src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/maintenance.html.twig

```diff
@@ -38,27 +37,9 @@
             </h3>
             <div class="card-block row">
               <div class="card-text">
-                <div class="form-group row">
-                  {{ ps.label_with_help(('Enable Shop'|trans), ('Activate or deactivate your shop (It is a good idea to deactivate your shop while you perform maintenance. Please note that the webservice will not be disabled).'|trans({}, 'Admin.Shopparameters.Help'))) }}
-                  <div class="col-sm">
-                    {{ form_errors(generalForm.enable_shop) }}
-                    {{ form_widget(generalForm.enable_shop) }}
-                  </div>
-                </div>
-                <div class="form-group row">
-                  {{ ps.label_with_help(('Maintenance IP'|trans), ('IP addresses allowed to access the front office even if the shop is disabled. Please use a comma to separate them (e.g. 42.24.4.2,127.0.0.1,99.98.97.96).'|trans({}, 'Admin.Shopparameters.Help'))) }}
-                  <div class="col-sm">
-                    {{ form_errors(generalForm.maintenance_ip) }}
-                    {{ form_widget(generalForm.maintenance_ip, { 'currentIp': currentIp}) }}
-                  </div>
-                </div>
-                <div class="form-group row">
-                  {{ ps.label_with_help(('Custom maintenance text'|trans), ('Custom text displayed on maintenance page while shop is deactivated.'|trans({}, 'Admin.Shopparameters.Help'))) }}
-                  <div class="col-sm">
-                    {{ form_errors(generalForm.maintenance_text) }}
-                    {{ form_widget(generalForm.maintenance_text) }}
-                  </div>
-                </div>
+                {{ ps.multistore_config_row(isMultishopConfigEnabled, generalForm.enable_shop) }}
+                {{ ps.multistore_config_row(isMultishopConfigEnabled, generalForm.maintenance_ip, { 'currentIp': currentIp}) }}
+                {{ ps.multistore_config_row(isMultishopConfigEnabled, generalForm.maintenance_text) }}
```

Comment:
I should find a way to get the `isMultishopConfigEnabled` from the macro without passing a parameter

Reply:
I'm not even sure you need this, maybe you can simply rely on the presence of the multistore checkbox field